src/main/java/com/conveyal/datatools/editor/controllers/api/ScheduleExceptionController.java

```diff
@@ -180,7 +175,21 @@ public static Object updateScheduleException (Request req, Response res) {
         }
         return null;
     }
-    
+
+    private static void checkDuplicateDates(ScheduleException ex, Collection<ScheduleException> exceptions) {
+        if (ex.dates != null) {
+            for (LocalDate date : ex.dates) {
+                // This is inefficient, but it keeps us from using the scheduleExceptionCountByDate map
+                // which had significant issues with mapdb corruption for some reason.
```

Comment:
We should summarize anything we know that could be a clue as to why this was being corrupted. I think what's very different about this map is that it's created with `BindUtils.multiHistogram` which I believe is making an auto-updated view or triggers or something. It's likely that there's some bug in combining these views with transactions.